index.bs

```diff
@@ -1179,6 +1187,14 @@ An {{MLOperand}}'s <dfn for=MLOperand>dataType</dfn> is its {{MLOperand/[[descri
 
 Since the {{MLOperand/[[builder]]}} object is bound by the {{MLGraphBuilder/constructor()}} constructor to an {{MLContext}} object, an {{MLOperand}} is also always bound to the same {{MLContext}} object.
 
+{{MLOperatorOptions}} has the following members:
+<dl dfn-type=dict-member dfn-for=MLOperatorOptions>
+    : <dfn>label</dfn>
+    ::
+        Optionally provided when an [=operator=] is created using {{MLGraphBuilder}} methods that create {{MLOperand}}s. The implementation may use this value to initialize the [=operator=]'s [=operator/label=].
```

Comment:
We don't currently say what the *default* is for a label reported in an error message if no label was given. There was chatter in https://github.com/webmachinelearning/webnn/issues/585 about automatically generating them based on the operator name and node index, and I don't think we settled on any particular algorithm yet, but should we explicitly mention for now that the label is empty "" if not set?

Reply:
For the dictionary member, it's provided automagically by Bikeshed in the rendered text: _label, of type USVString, defaulting to ""_

If you mean the label concept of an operator, the spec intentionally doesn't say anything specific about how the passed string is used. It could be ignored, replaced, truncated, augmented, etc.